### ISSUE TITLE
`ref` modifier appears to be removed from `struct` types

### ISSUE THREAD

**Issue by BillWagner** [mono/api-doc-tools#260]
See more details in dotnet/docs#5208

There are couple important points here:

- For types (like `Span<T>`) that are `ref struct` types, the `ref` modifier is missing.
- When a method signature returns a `ref T`, the `ref T` modifier is missing.

**Comment by BillWagner** [comment]
A similar problem happens for `ref readonly struct` declarations.

**Comment by joelmartinez** [comment]
For the second point, that will be addressed in the next release by the resolution to https://github.com/mono/api-doc-tools/issues/249

**Comment by mairaw** [comment]
@joelmartinez should I add a new issue or just include the conditions from https://github.com/MicrosoftDocs/feedback/issues/468 here too given this one is still open?

**Comment by joelmartinez** [comment]
@mairaw Yeah, a new issue would be ideal, so we can keep the scope of each issue a bit more focused. This one will be for the `ref struct`, and the new one will address the indexer ref scenario

**Comment by mairaw** [comment]
OK, filed a new one for the indexers then: https://github.com/mono/api-doc-tools/issues/422

**Comment by mimisasouvanh** [comment]
Internal [bug](https://ceapex.visualstudio.com/Engineering/_workitems/edit/167407) logged.
